psi4/src/psi4/scfgrad/jk_grad.cc

```diff
@@ -390,7 +390,7 @@ void DFJKGrad::build_Amn_terms()
         // > Integrals < //
         int nthread_df = df_ints_num_threads_;
 #pragma omp parallel for schedule(dynamic) num_threads(nthread_df)
-        for (long int PMN = 0L; PMN < NP * npairs; PMN++) {
+        for (long int PMN = 0L; PMN < (long) NP * npairs; PMN++) {
```

Comment:
Did we want to migrate all of these to `static_cast<long>(NP)` like the others?

Reply:
for new casts, yes, I agree.

Reply:
updated.